policy-controller/k8s/status/src/index.rs

```diff
@@ -158,7 +164,7 @@ impl Index {
 
     // If the route is new or its parents have changed, return true so that a
     // patch is generated; otherwise return false.
```

Comment:
```suggestion
    // If the route is new or its parentRefs and/or backendRefs have changed, 
    // return true, so that a patch is generated; otherwise return false.
```